Resources/Prototypes/_Goobstation/Entities/Objects/Tools/jetpacks.yml

```
@@ -33,3 +33,44 @@
+      weightlessAcceleration: 0.8
+      weightlessFriction: 1
+      weightlessModifier: 0.8
+
+# Filled improvised - GoobStation
+- type: entity
+  id: JetpackImprovisedFilled
+  parent: JetpackImprovised
+  name: improvised jetpack
+  suffix: Filled
+  components:
+  - type: GasTank
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)